include/Graphs/PAG.h

```diff
@@ -144,11 +144,16 @@ class PAG : public GenericGraph<PAGNode,PAGEdge>
     }
 
     /// Singleton design here to make sure we only have one instance during any analysis
+    /// Sometimes we want to check if PAG has been made, but not make it if it has not.
+    /// In that case use skipCreate = true.
     //@{
-    static inline PAG* getPAG(bool buildFromFile = false)
+    static inline PAG* getPAG(bool buildFromFile = false, bool skipCreate = false)
     {
         if (pag == nullptr)
         {
+            if (skipCreate) {
+                return nullptr;
+            }
```

Comment:
This is only used when dumping ICFG. I suggest making an option rather than changing this code fragment.

Reply:
As suggested above, can I revert this change, but add a separate method to get the PAG if available, for the sake of the dumper?